### PR TITLE
WEBEU3 - Preprocessing-I - Evans Ibok

### PR DESCRIPTION
**Pull request by evansibok**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,144 @@
...
 <head>
-  <meta charset="utf-8">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<title>My Resume</title>
 
-  <title>My Resume</title>
+	<!-- Google Fonts-->
+	<link href="https://fonts.googleapis.com/css?family=Dosis|Nunito&display=swap" rel="stylesheet">
```
Comment (OloruntobiAwoderu) [inline index.html:12]: You Incorporated google fonts into your project. Very Nice!!!

File: less/index.less [hunk 2]
```diff
@@ -54,6 +54,397 @@ table {
...
+	height: 100%;
 }
 
+/* Custom CSS */
+
+
+@primaryColor: #F55C42;
+@baseColor: rgb(29, 29, 29);
+@baseFont: 'Nunito', 'sans-serif';
+@sectionHFiveColor: rgba(29, 29, 29, 0.5);
```
Comment (OloruntobiAwoderu) [inline less/index.less:147]: Nice use of variables

File: less/index.less [hunk 3]
```diff
@@ -54,6 +54,397 @@ table {
...
+}
+
+header {
+	.flex-column();
+	padding: 2rem;
+
+	.brand-info {
+		.flex-space-between();
+
+		@media screen and (max-width: 500px) {
```
Comment (OloruntobiAwoderu) [inline less/index.less]: You could have also set max-width:500px as a variable since you are using it multiple times

File: less/index.less [hunk 4]
```diff
@@ -54,6 +54,397 @@ table {
...
+					line-height: 1.1rem;
+				}
+			}
+
+			.admin-olotu {
+				margin-top: 2rem;
+
+				h3 {
+					margin-bottom: 0.5em;
+
```
Comment (OloruntobiAwoderu) [inline less/index.less:386]: Nice use of Nesting!!!

**Review by OloruntobiAwoderu — COMMENTED** [review]
I can see you commit your node-modules. You should create a gitignore file and add node-modules to it so you don't commit it to your repository. If you don't know how to do this, you can let me know so i can help you

File: less/index.less [hunk 5]
```diff
@@ -54,6 +54,397 @@ table {
...
+	max-width: 1024px;
+	background-color: rgb(255, 255, 255);
+
+	@media screen and (max-width: 500px) {
+		max-width: 450px;
+	}
+}
+
+header {
+	.flex-column();
```
Comment (OloruntobiAwoderu) [inline less/index.less:193]: Nice use of mixins

**Review by OloruntobiAwoderu — COMMENTED** [review]
Great job on this project overall Evans!!! I really liked your design and you completed all MVP requirements except having a navigation bar with four elements. You also completed a stretch goal. Try and format your code after completing your work to make it more easier to read. You can look up the prettier extension on VsCode. Keep up the good work!!!

File: less/index.less (hunk 3 above)
Comment (evansibok) [reply less/index.less]: I have set the mobile screen size as a variable and used it throughout the project.

**Comment by evansibok** [comment]
Thank you for the commendation.  I have removed the node_modules from my commits. Do I have to complete the navigation bar with all four elements before my project is merged?

**Comment by evansibok** [comment]
Hello @OloruntobiAwoderu I have added the navigations to both the desktop and mobile view.

**Pull request merged**